src/components/SelectionList/Search/DateCell.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useMemo} from 'react';
+import type {TaskListItemType, TransactionListItemType} from '@components/SelectionList/types';
+import TextWithTooltip from '@components/TextWithTooltip';
+import useThemeStyles from '@hooks/useThemeStyles';
+import DateUtils from '@libs/DateUtils';
+import {isTaskListItemType} from '@libs/SearchUIUtils';
+import {getCreated as getTransactionCreated} from '@libs/TransactionUtils';
+import CONST from '@src/CONST';
+
+type DateCellProps = {
+    item: TransactionListItemType | TaskListItemType;
+    showTooltip: boolean;
+    isLargeScreenWidth: boolean;
+};
+
+function DateCell({item, showTooltip, isLargeScreenWidth}: DateCellProps) {
```

Comment:
Instead of passing the item, can we just pass `created` directly? That way this component doesn't need to handle business logic related to the type of item we're passing